src/server/redis_connection.h

```diff
@@ -90,6 +90,9 @@ class Connection {
   uint32_t GetPort() { return port_; }
   void SetListeningPort(int port) { listening_port_ = port; }
   int GetListeningPort() { return listening_port_; }
+  void SetAnnounceIP(std::string ip) { announce_ip_ = std::move(ip); }
+  std::string GetAnnounceIP() { return announce_ip_; }
```

Comment:
There seems to be no place to use `GetAnnounceIP()`, and I personally don't feel the need to add this function

Reply:
done